src/audio/google_rtc_audio_processing.c

```diff
@@ -36,6 +36,8 @@
 #include <google_rtc_audio_processing_platform.h>
 
 #define GOOGLE_RTC_AUDIO_PROCESSING_SAMPLERATE 48000
+#define GOOGLE_RTC_AUDIO_PROCESSING_NUM_AEC_REFERENCE_CHANNELS 2
+#define GOOGLE_RTC_AUDIO_PROCESSING_NUM_CAPTURE_CHANNELS 1
```

Comment:
do we really want to hardcode this? I would think maybe we want to make this a tuple in the topology, even the framerate really is questionable being hardcoded long term

Reply:
Yes, I agree that is something we should do instead, but this CL is a step towards that.
This change removes the dependence on the values hardcoded in the lib.